modules/client-configuration/pages/registration-overview-debian.adoc

```diff
@@ -0,0 +1,17 @@
+[[debian-registration-overview]]
+= Debian Client Registration
+
+You can register {debian} clients to your {productname} Server.
+The method and details varies depending on the operating system of the client.
```

Comment:
I do not understand this.  At this point it seems clear that you are talking about Debian.

Reply:
It's what all of them say. I see your point, but it would need to be edited globally.